elixir/apps/api/lib/api/controllers/actor_group_json.ex

```diff
@@ -0,0 +1,35 @@
+defmodule API.ActorGroupJSON do
+  alias Domain.Actors
+  alias Domain.Repo.Paginator.Metadata
+
+  @doc """
+  Renders a list of Actor Groups.
+  """
+  def index(%{actor_groups: actor_groups, metadata: metadata}) do
+    %{data: for(actor_group <- actor_groups, do: data(actor_group))}
+    |> Map.put(:metadata, metadata(metadata))
```

Comment:
This might be a bit easier to read:
```suggestion
    %{data: Enum.map(actor_groups, &data/1), metadata: metadata(metadata)}
```